tests/testswitch.nim

```diff
@@ -2,23 +2,7 @@ import unittest, tables
 import chronos
 import chronicles
 import nimcrypto/sysrand
-import ../libp2p/[switch,
-                  multistream,
-                  protocols/identify,
-                  connection,
-                  transports/transport,
-                  transports/tcptransport,
-                  multiaddress,
-                  peerinfo,
-                  crypto/crypto,
-                  peer,
-                  protocols/protocol,
-                  muxers/muxer,
-                  muxers/mplex/mplex,
-                  muxers/mplex/types,
-                  protocols/secure/secio,
-                  protocols/secure/secure,
-                  stream/lpstream]
+import ../libp2p/libp2p
```

Comment:
Or `../libp2p/all` to avoid the stutter.

Reply:
I'm trying to export it through the main `libp2p` file in the root of the project, but doesn't seem to work - https://github.com/status-im/nim-libp2p/pull/133#pullrequestreview-389542329